packages/server/src/requestPipeline.ts

```diff
@@ -316,12 +317,7 @@ export async function processGraphQLRequest<TContext>(
       persistedQueryCache.set(
         queryHash,
         query,
-        config.persistedQueries &&
-          typeof config.persistedQueries.ttl !== 'undefined'
-          ? {
-              ttl: config.persistedQueries.ttl,
-            }
-          : Object.create(null),
+        config.persistedQueries?.ttl ?? 0,
```

Comment:
I think maybe fall back to undefined? which a vague glance suggests means "use the cache's default ttl"? Double check me on this though.

Reply:
Yeah, undefined falls back to `Keyv.opts.ttl` or `null`.
c4f7e7f